admin-ui/app/utils/AppAuthProvider.js

```diff
@@ -41,8 +41,7 @@ class AppAuthProvider extends Component {
       console.warn('Parameters to process authz code flow are missing.')
       return
     }
-    return `${authzBaseUrl}?response_type=${responseType}
-     &redirect_uri=${redirectUrl}&client_id=${clientId}&scope=${scope}&state=${state}&nonce=${nonce}`
+    return `${authzBaseUrl}?response_type=${responseType}&redirect_uri=${redirectUrl}&client_id=${clientId}&scope=${scope}&state=${state}&nonce=${nonce}`;
```

Comment:
This line should  be formatted, i mean the whole file

Reply:
With formatter it gives the same result.